ksql-engine/src/main/java/io/confluent/ksql/planner/plan/KsqlBareOutputNode.java

```diff
@@ -39,7 +39,17 @@ public KsqlBareOutputNode(
       final OptionalInt limit,
       final TimestampExtractionPolicy extractionPolicy
   ) {
-    super(id, source, schema, limit, extractionPolicy);
+    super(
```

Comment:
Don't think adding this was intentional....  we shouldn't be doing this.

Reply:
I was just hacking it to make it work - wasn't going to commit until your fix is in :)